website/_kb/user-guide/querying/favorites-tagging.md

```diff
@@ -0,0 +1,22 @@
+---
+category: querying
+parent_category: user-guide
+helpscout_url: https://help.redash.io/article/45-query-filters
+title: Favorites & Tagging
+slug: favorites-tagging
+---
+
+Redash users write a lot of queries and dashboards! Favorites and Tagging are here to make finding them easy as your list of queries grows from a few hundred to a few thousand.
+
+<img src="/assets/images/docs/gitbook/favorites-example.png" width="100%">
+
+## Favorites
+You can favorite a dashboard or query by clicking the star to the left of its title anywhere in Redash. The star will turn gold to indicate success. Your list of favorites of each kind can be viewed from the query or dashboard list view. Just click the `Favorites` filter on the right.
+
+
+## Tagging
+You can tag queries by subject matter, location, user or any parameter that is meaningful to your organization. Tags are added from the query editor or the dashboard editor. Hover your mouse on the query or dashboard title and an `+Add Tag` button will appear. In the modal that appears you can select as many tags as you need. The modal will suggest previously-used tags as you type. Hit `Save` when you're finished or `Esc` to abort tagging.
```

Comment:
Worth adding a callout mentioning that it's good idea to have an internal discussion on the taxonomy they want to create with tags.